sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/BeamSqlSeekableTable.java

```diff
@@ -28,8 +29,12 @@
  * FROM FACT_TABLE JOIN LOOKUP_TABLE ON ...}.
  */
 public interface BeamSqlSeekableTable extends Serializable {
-  /** prepare the instance. */
-  default void setUp() {}
+  /**
+   * prepare the instance
```

Comment:
there is a checkstyle error: eckstyle] [ERROR] /runner/_work/beam/beam/sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/BeamSqlSeekableTable.java:32: First sentence should end with a period. [JavadocStyle]
```suggestion
   * prepare the instance.
```